test/powershell/Language/Parser/LanguageAndParser.TestFollowup.Tests.ps1

```diff
@@ -303,3 +303,9 @@ Describe "Hash expression with if statement as value" -Tags "CI" {
         $hash['h'] | Should -BeExactly 'h'
     }
 }
+
+Describe "Hashtable is case insensitive" -Tag CI {
+    It "When LANG is C.UTF-8" -Skip:($IsWindows) {
+        sh -c 'LANG=C.UTF-8 pwsh -c ''$h=@{h=1};$h.H''' | Should -Be 1
```

Comment:
Minor nit: using the `-command` parameter on `pwsh` would be slightly more readable and using a different letter for the hashtable property.
```suggestion
        sh -c 'LANG=C.UTF-8 pwsh -command ''$h=@{p=1};$h.P''' | Should -Be 1
```